src/interfaces/importhelpers.cpp

```diff
@@ -602,17 +602,20 @@ namespace
         uint32_t publicKeyLength;
         if (1 != md_get_column_value_as_blob(sourceAssembly, mdtAssembly_PublicKey, 1, &publicKey, &publicKeyLength))
             return E_FAIL;
-        
+
+        span<const uint8_t> publicKeyTokenSpan;
         StrongNameToken publicKeyToken;
         if (publicKey != nullptr)
         {
             assert(IsAfPublicKey(flags));
             flags &= ~afPublicKey;
             RETURN_IF_FAILED(StrongNameTokenFromPublicKey({ publicKey, publicKeyLength }, publicKeyToken));
+            publicKeyTokenSpan = { publicKeyToken.data(), publicKeyToken.size() };
         }
         else
         {
             assert(!IsAfPublicKey(flags));
+            publicKeyTokenSpan = { nullptr, 0 };
```

Comment:
```suggestion
```

This isn't needed. The `span<T>` has a default constructor that initializes it to this state.